### ISSUE TITLE
Show login mask on direct ajax.php link

### ISSUE THREAD

**Issue by perryflynn** [perryflynn/webshelf-file-explorer#38]
When download link clicked without logged in user, show a login mask.

**Comment by perryflynn** [comment]
Add a view to controller system.
There is no other chance to add this...
